algebird-core/src/main/scala/com/twitter/algebird/Operators.scala

```diff
@@ -16,23 +16,36 @@ limitations under the License.
 package com.twitter.algebird
 
 object Operators {
-  implicit def toPlus[T: Semigroup](t: T): PlusOp[T] = new PlusOp(t)
-  implicit def toMinus[T: Group](t: T): MinusOp[T] = new MinusOp(t)
-  implicit def toTimes[T: Ring](t: T): TimesOp[T] = new TimesOp(t)
+  @deprecated("use Operators.Ops", "0.13.8")
+  def toPlus[T: Semigroup](t: T): PlusOp[T] = new PlusOp(t)
+  @deprecated("use Operators.Ops", "0.13.8")
+  def toMinus[T: Group](t: T): MinusOp[T] = new MinusOp(t)
+  @deprecated("use Operators.Ops", "0.13.8")
+  def toTimes[T: Ring](t: T): TimesOp[T] = new TimesOp(t)
+
   implicit def toRichTraversableFromIterator[T](t: Iterator[T]): RichTraversable[T] =
     new RichTraversable(t)
   implicit def toRichTraversable[T](t: Traversable[T]): RichTraversable[T] =
     new RichTraversable(t)
+
+  implicit class Ops[A](private val a: A) extends AnyVal {
```

Comment:
this pattern is better because it doesn't have to allocate. The `PlusOp` etc pattern has to allocate because it captures the typeclass in the constructor vs the method callsite.